themes/makedeb/assets/scss/post.scss

```diff
@@ -1,75 +1,157 @@
 @import "variables";
 
 .post {
-	padding: 3em 5%;
-	color: $dark-grey;
-	
-	.title, .date {
-		text-align: center;
-	}
-
-	.title {
-		font-size: 2.75em;
-	}
-
-	.date {
-		font-size: 1.1em;
-		line-height: 1.6em;
-	}
-
-	.content {
-		padding-top: 2em;
-		max-width: 50em;
-		margin-left: auto;
-		margin-right: auto;
-
-		h2 {
-			font-size: 2em;
-			margin: 0.7em 0 0.3em;
-		}
-
-		h3 {
-			font-size: 1.6em;
-			margin: 0.7em 0 0.4em;
-		}
-
-		p, li {
-			font-size: 1.2em;
-			line-height: 1.3em;
-		}
-
-		p {
-			padding-bottom: 1em;
-		}
-		
-		ul {
-			padding-bottom: 1em;
-		}
-
-		li {
-			padding-left: 2em;
-		}
-	}
-	
-	.closing-hr {
-		width: 15em;
-	}
-
-	.author {
-		display: flex;
-		flex-direction: column;
-		align-items: center;
-
-		img {
-			height: 5em;
-			width: 5em;
-			border-radius: 50%;
-			margin: 1em 0;
-		}
-
-		.author-name {
-			font-size: 1.5em;
-		}
-
-	}
+  padding: 3em 5%;
+  @extend .themeBackgroundColor;
+
+  .title,
+  .date {
+    text-align: center;
+  }
+
+  .title {
+    font-size: 2.75em;
+  }
+
+  .date {
+    font-size: 1.1em;
+    line-height: 1.6em;
+  }
+
+  hr {
+    @extend .themePostHrColor;
+  }
+  hr {
+    @extend .themePostHrColor;
+  }
```

Comment:
This duplication isn't needed, right? Also I think we could just remove it entirely and apply this styling to all `<hr>` in `main.scss`, as places like `/donate` have the `<hr>` element a bit too bright on the dark theme.